ndcube/tests/test_sequence_plotting.py

```diff
@@ -253,14 +257,14 @@
     seq.cube_like_dimensions, unit=u.pix).value.astype(int)
 cube_like_new_x_axis_coords2_shape[-1] = 1
 cubelike_none_axis_ranges_axis2 = [
-    np.arange(0, int(seq.cube_like_dimensions[0].value)+1), np.arange(0, 4),
+    np.arange(0, int(seq.cube_like_dimensions[0].value)), np.arange(0, 3),
```

Comment:
Even though I have changed the values of the expected object as it was required in, I need to cross-check with @DanRyanIrish for contextual meaning of the values.

Reply:
This is correct.